backend/inspirehep/search/api.py

```diff
@@ -387,13 +387,13 @@ def query_from_iq(self, query_string):
         """
         if not is_superuser_or_cataloger_logged_in():
             if not query_string:
-                user_query = Q("term", status="open")
+                user_query = Q("terms", status=["open", "closed"])
```

Comment:
we need by default to display only open, do we have to change it?

Reply:
yes we do https://github.com/inspirehep/inspirehep/pull/2685/files#diff-2bbc000ac49ab94dc52c7931324b1490f0ab382bc8f67e9da203b444d0c6d8e1R71-R74